safe-global/safe-wallet-web
# Feat: add swaps card to safe apps list

## What it solves

Resolves https://www.notion.so/safe-global/Add-a-block-to-apps-page-985e220c975b42b7950929ecb02a6715?pvs=4

## How this PR fixes it
- Add a component for showing native swaps in the safe apps list
- Allow the card to be dismissed. This decision is persisted in localstorage.
- Make the cards respond to the search and filter like the other safe apps.

## How to test it
- Open the safe apps list.
- By default the new swaps card should be shown first in the list
- It should be shown when searching with the keyword `swap`, along with other swapping apps.
- It should show when filtering for the `defi` category
- Clicking `try now` should bring you to the swaps feature
- Clicking `Don't show` should remove the card from the list. It will remain hidden unless localstorage is cleared.


## Screenshots
<img width="1269" alt="image" src="https://github.com/safe-global/safe-wallet-web/assets/17801424/604be7ab-76a8-4b65-8f11-5d67e39b7579">


## Checklist
